pyperf/_utils.py

```diff
@@ -266,8 +266,10 @@ def create_environ(inherit_environ, locale, copy_all):
     if copy_all:
         return os.environ
     env = {}
-
-    copy_env = ["PATH", "PYTHONPATH", "HOME", "TEMP", "COMSPEC", "SystemRoot", "SystemDrive"]
+    copy_env = ["PATH", "HOME", "TEMP", "COMSPEC", "SystemRoot", "SystemDrive"]
+    # TODO: In the future, maybe we should manage CPython environment variables
+    # depending on the Python version.
+    copy_env.extend(["PYTHONPATH", "PYTHON_CPU_COUNT", "PYTHON_GIL"])
```

Comment:
Just put it directly in the list, no?

Reply:
Well let's separate CPython specific variables :)

Reply:
You can separate them with a comment and/or a newline.

Reply:
Well, I am not sure why you want to maintain the single list declaration even if this is not a performance-sensitive code